assets/section-main-product.css

```diff
@@ -383,6 +405,13 @@ a.product__text {
   }
 }
 
+@media screen and (min-width: 990px) {
+  .different-color-scheme {
+    padding: 3rem;
+    margin-top: 0rem;
```

Comment:
```suggestion
```

We can also omit this as Ludo said since you're already applying it here:

https://github.com/Shopify/dawn/blob/73991f6a031a4443cf75b7f559e33f13e4af1dd6/assets/section-main-product.css#L389-L393